src/odemis/driver/andorcam2.py

```diff
@@ -3031,6 +3154,9 @@ def AbortAcquisition(self):
         self.status = AndorV2DLL.DRV_IDLE
         self.acq_aborted.set()
 
+    def GetOldestImage16(self, cbuffer, size):
+        return self.GetMostRecentImage16(cbuffer, size)
```

Comment:
GetOldestImage16 uses the call to mostRecentImage without any minus, inverse or something to annotate that it is actually not getting the most recent image? How does that work?

Reply:
They behave the same because the simulator just generate one image on the fly, there is no image buffer. I'm added a comment.